sdk/monitor/Azure.Monitor.Ingestion/src/LogsIngestionClient.cs

```diff
@@ -112,7 +140,7 @@ internal static IEnumerable<BatchedLogs<T>> Batch<T>(IEnumerable<T> logEntries)
                     WriteMemory(stream, memory);
                     if ((entryCount + 1) == logEntries.Count())
                     {
-                        // reached end of logEntries and we haven't returned yet
+                        // reached end of logs and we haven't returned yet
```

Comment:
I'm curious why we are writing the JSON ourselves rather than using the System.Text.Json primitives like [Utf8JsonWriter](https://docs.microsoft.com/en-us/dotnet/api/system.text.json.utf8jsonwriter?view=net-6.0). Doesn't need to be addressed as part of this PR, but I'm interested in whether we can change the implementation so that we don't have to write the array ourselves.

Reply:
I will make a note to address this before GA.